maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Handlers/TouchRouteHandler.swift

```diff
@@ -11,21 +11,34 @@ struct TouchRouteHandler: HTTPHandler {
     
     func handleRequest(_ request: FlyingFox.HTTPRequest) async throws -> FlyingFox.HTTPResponse {
         let decoder = JSONDecoder()
+      
+        let orientation = XCUIDevice.shared.orientation
         
+        logger.info("XCUIDevice.shared.orientation: \(String(describing: orientation))")
+      
         guard let requestBody = try? decoder.decode(TouchRequest.self, from: request.body) else {
             return AppError(type: .precondition, message: "incorrect request body provided for tap route").httpResponse
         }
         
+        let (width, height) = ScreenSizeHelper.physicalScreenSize()
+        
+        let (x, y) = switch (orientation) {
+        case .portrait: (requestBody.x, requestBody.y)
+        case .landscapeLeft: (width - requestBody.y, requestBody.x)
+        case .landscapeRight: (requestBody.y, height - requestBody.x)
+        default: fatalError("Not implemented yet")
```

Comment:
nit: indentation

Reply:
I'm honeslty not sure what's the right indentation here, I haven't been writing much Swift recently.

I created https://github.com/mobile-dev-inc/maestro/issues/1827 to make this situation better in the future:)